src/color.js

```diff
@@ -971,7 +1019,7 @@ Object.assign(Color, {
 	defaults: {
 		gamutMapping: "lch.chroma",
 		precision: 5,
-		deltaE: "76", // Default deltaE method
+		deltaE: { method: "76" }, // Default deltaE method
```

Comment:
I'm not necessarily opposed to this change, but I would like some justification for this change (and ideally this should have been a separate PR as it does not seem to be a required change for HyAB distance).